docs/source/install.rst

```diff
@@ -1,97 +1,94 @@
 .. _install:
 
-
 ============
 Installation
 ============
 
-These installation instructions explain how to install the Jupyter Notebook and
-the IPython kernel.
-
-While Jupyter allows users to run code in many different
-programming languages, the Jupyter Notebook itself is implemented in Python.
-To install Jupyter Notebook, you will also need Python installed on your system.
+This section explains how to install the Jupyter Notebook and the IPython
+kernel.
 
-**Installation Scenarios**
+While Jupyter runs code in many different programming languages, Python is a
+prerequisite for installing Jupyter notebook. Select one of the scenarios to
+install Jupyter:
 
-* :ref:`New user: New to Python and Jupyter <new-to-python-and-jupyter>`
-* :ref:`Experienced Python user: New to Jupyter <existing-python-new-jupyter>`
-* :ref:`Upgrading Jupyter <upgrading>`
+    - :ref:`Jupyter (new to Python) <new-to-python-and-jupyter>`
+    - :ref:`Jupyter (experience with Python) <existing-python-new-jupyter>`
```

Comment:
(In case it's not clear, the link text here doesn't technically have to match the titles it links to - though of course it helps the user if they're similar)